chimg/chroot.py

```diff
@@ -288,6 +292,33 @@ def _file_install(self, f: Dict):
             os.chown(f_path, -1, f["group"])
         if f.get("mode", None):
             os.chmod(f_path, f["mode"])
+        logger.info(f"Created file {f_path} using provided file content")
+
+    def _local_file_upload(self, f: Dict):
```

Comment:
can you add unittests (in addition to the functional tests - thanks for those!) please?